api/comment_threads.rb

```diff
@@ -78,6 +78,7 @@
   comment.anonymous_to_peers = bool_anonymous_to_peers || false
   comment.author = user
   comment.comment_thread = thread
+  comment.children_count = 0
```

Comment:
Or are you suggesting this because we should use `<singular_noun>_count` for count fields?